### PR TITLE
CORCI-1103 Build: Add CentOS 8.4 and Leap 15.3

### PR DESCRIPTION
**Pull request by JohnMalmberg**
Signed-off-by: John E Malmberg <john.e.malmberg@intel.com>

### CODE REVIEW

File: packaging/Dockerfile.mockbuild [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2018-2019, Intel Corporation
+# Copyright 2018-2020, Intel Corporation
```
Comment (jolivier23) [inline packaging/Dockerfile.mockbuild]: 2020?

**Review by jolivier23 — CHANGES_REQUESTED** [review]

File: packaging/Dockerfile.mockbuild (hunk 1 above)
Comment (JohnMalmberg) [reply packaging/Dockerfile.mockbuild]: Need to change that in the packaging repo and then redo all of these PRs in progress.

Comment (jolivier23) [reply packaging/Dockerfile.mockbuild]: so are you planning to update this?

**Review by jolivier23 — CHANGES_REQUESTED** [review]

**Comment by JohnMalmberg** [comment]
Obsolete

**Pull request closed without merging**